android/src/main/kotlin/hk/gogovan/flutter_label_printer/FlutterLabelPrinterPlugin.kt

```diff
@@ -30,11 +30,11 @@ class FlutterLabelPrinterPlugin : FlutterPlugin, ActivityAware {
         methodHandler = FlutterLabelPrinterMethodHandler(context, bluetoothSearcher)
 
         channel =
-            MethodChannel(flutterPluginBinding.binaryMessenger, "com.gogovan/flutter_label_printer")
+            MethodChannel(flutterPluginBinding.binaryMessenger, "hk.gogovan.label_printer.flutter_label_printer")
```

Comment:
the app bundle need to be `hk.gogovan.label_printer` in order to use this plugin? 🤔 correct me if I'm wrong

Reply:
no, this is just for namespacing purposes. We can use any string for this.

Reply:
got it!